lib/xcake/xcode/scheme_list.rb

```diff
@@ -78,6 +80,31 @@ def create_schemes_for_application(target)
         end
       end
 
+      # Creates schemes based on a framework target
+      #
+      # @param    [Target] target
+      #           target to create framework schemes for
+      #
+      def create_schemes_for_application(target)
+        target.build_configurations.each do |c|
+          scheme = Scheme.new
+
+          scheme.name = "#{target.name}-#{c.name}"
+          scheme.add_build_target(target)
+          @xcschememanagement['SuppressBuildableAutocreation'][target.uuid] = {"primary" => true}
+
+          unit_test_target = project.find_unit_test_target_for_target(target)
+
+          if unit_test_target then
+            scheme.add_test_target(unit_test_target)
+            unit_test_target.add_dependency(target)
+            @xcschememanagement['SuppressBuildableAutocreation'][unit_test_target.uuid] = {"primary" => true}
```

Comment:
Line is too long. [109/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.